src/call/client.rs

```diff
@@ -331,6 +331,15 @@ impl<Req> StreamingCallSink<Req> {
         }
     }
 
+    /// By default the sink works in normal mode, that is, `start_send` always starts to send message
+    /// immediately. But when the `enhance_batch` is enabled, the stream will be batched together as
+    /// much as possible. The specific rule is listed below:
+    /// Set the `buffer_hint` of the non-end message in the stream to true, and set the `buffer_hint` of
+    /// the last message to false in `poll_flush`, so that the previously bufferd messages will be sent out.
```

Comment:
The implement doesn't match the description. The implement sets buffer hint to false only when there is at least a false flag.

Reply:
Fixed